docs/ref/index.txt

```diff
@@ -31,3 +31,4 @@ API Reference
    utils
    validators
    views
+   paginator
```

Comment:
It looks like this list is supposed to be in alphabetical order ("Built-in Views" violates this, but it looks like a mistake). I suggest you move this to the correct place.